### PR TITLE
 feat(apm): add satispay logo and updated payu logo

### PR DESCRIPTION
**Pull request by JianyiGao**
Add satispay logo and updated payu logo
Bump sdk constant version, related pr https://github.com/paypal/paypal-sdk-constants/pull/104

https://engineering.paypalcorp.com/jira/browse/DTALTPAY-1115

Tested locally with checkout components
image

image

### CODE REVIEW

File: src/logos/satispay/logo.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import { node, type ComponentNode } from "@krakenjs/jsx-pragmatic/src";
+
+import { SVGLogo, getLogoColors, type SVGLogoProps } from "../../lib";
+import { LOGO_COLOR, LOGO } from "../../constants";
+import { type LogoColorMap } from "../../types";
+
+const LOGO_COLORS: LogoColorMap = {
+  [LOGO_COLOR.DEFAULT]: {
+    primary: "#2C2E2F",
```
Comment (ericyupaypal) [inline src/logos/satispay/logo.jsx:12]: Based on the UX design, the default logo could should be white rendered on the red background. The way you have it configured shows a dark color logo on the default red button background 
```suggestion
    primary: "#FFFFFF",
```
Screen Shot 2022-11-01 at 5 35 25 PM

File: src/logos/payu/logo.jsx [hunk 2]
```diff
@@ -9,22 +9,16 @@ import { type LogoColorMap } from "../../types";
...
     secondary: "#FFFFFF",
-    tertiary: "#000000",
-    quaternary: "#FFFFFF",
   },
   [LOGO_COLOR.BLACK]: {
-    primary: "#333030",
-    secondary: "#333030",
-    tertiary: "#ffffff",
-    quaternary: "#333030",
+    primary: "#A6C307",
```
Comment (ericyupaypal) [inline src/logos/payu/logo.jsx:20]: Based on the UX design, the logo rendered on the white and silver buttons should be dark. The way you have it configured shows the default branded color.
```suggestion
    primary: "#2C2E2F",
    secondary: "#2C2E2F"
```
Screen Shot 2022-11-01 at 5 39 17 PM

**Review by ericyupaypal — CHANGES_REQUESTED** [review]

**Review by ericyupaypal — APPROVED** [review]
Appproved

**Review by dtjones404 — APPROVED** [review]
lgtm!

**Pull request merged**